contracts/DifficultyOracle.sol

```diff
@@ -19,7 +30,11 @@ contract DifficultyOracle {
         questionId = _questionId;
     }
 
+    /// @dev Triggers an oracle report. If the transaction occurs within the prescribed valid time window, the oracle will attempt to report a result to the prediction market system. The result reported will be two EVM words (2*32=64 bytes). If the witnessed difficulty exceeds the target difficulty, the first word will be 0 and the second will be 1. Otherwise, if the witnessed difficulty is less than or equal to the target difficulty, the first word will be 1 and the second will be 0. (AUDIT: this function's visibility can be limited to external)
```

Comment:
>(AUDIT: this function's visibility can be limited to external)

What would be the benefit?

Reply:
nm, https://medium.com/@gus_tavo_guim/public-vs-external-functions-in-solidity-b46bcf0ba3ac